src/Raven.Server/Documents/Queries/Results/TimeSeries/TimeSeriesRetriever.cs

```diff
@@ -75,7 +76,7 @@ public enum ResultType
             Aggregated
         }
 
-        public IEnumerable<DynamicJsonValue> InvokeTimeSeriesFunction(DeclaredFunction declaredFunction, string documentId, object[] args, out ResultType resultType)
+        public IEnumerable<DynamicJsonValue> InvokeTimeSeriesFunction(DeclaredFunction declaredFunction, string documentId, object[] args, out ResultType resultType, CancellationToken token = default)
```

Comment:
is there a place where you are not passing the token? maybe it does not require a default value?